{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/static/css/main_style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  padding: 0;
+  margin: 0;
+  background: #f2f6e9;
+}
+/*--- navigation bar ---*/
```

Comment:
Please delete unused code rather than commenting it.

Reply:
Removed it